src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -82,7 +82,7 @@
         </service>
 
         <service id="console.command.messenger_consume_messages" class="Symfony\Component\Messenger\Command\ConsumeMessagesCommand">
-            <argument /> <!-- Message bus locator -->
```

Comment:
https://github.com/symfony/symfony/blob/0b6a9f9058046a9bc1ef8b8f16c03ed96ee131fe/src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php#L263 must be removed

Reply:
good catch